django_cyverse_auth/views.py

```diff
@@ -119,7 +119,7 @@ def cas_callback_authorize(request):
                      " Check the CAS server and caslib.py for more"
                      " information.")
         # NOTE: Make sure this redirects the user OUT of the loop!
-        return login(request)
+        return django_login(request)
```

Comment:
login was not defined, thought there is a django_login imported at the top of this file.